test/edu/ucsb/nceas/metacat/dataone/D1AuthHelperTest.java

```diff
@@ -97,7 +114,8 @@ private SystemMetadata getGenericSysmetaObject() throws Exception {
         SystemMetadata sysmeta =
             TypeFactory.buildMinimalSystemMetadata(TypeFactory.buildIdentifier("dip"),
                                                    new ByteArrayInputStream(
-                                                       ("tra la la la la").getBytes("UTF-8")),
+                                                       ("tra la la la la").getBytes(
```

Comment:
it's good practice to give some indication of what this is, instead of a totally unrelated content. When debugging, it's easier to figure out where "test Sysmeta data inputstream" came from, versus "tra la la la la". (great tune, though)

Reply:
Understood, thank you and I've updated the string content.